From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Sat, 16 Jun 2018 00:41:29 -0400
Subject: Fix keyRefreshService-test.js -- service is on by default.

Since 37f708b24a53a1acd39d050d00f13b3baf80e130 turned the key refresh
service on by default, the unit tests should have been updated to
match, but it appears that they were not.
---
 package/tests/keyRefreshService-test.js | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/package/tests/keyRefreshService-test.js b/package/tests/keyRefreshService-test.js
index 76ceb7a..a2c66e9 100644
--- a/package/tests/keyRefreshService-test.js
+++ b/package/tests/keyRefreshService-test.js
@@ -214,12 +214,12 @@ test(withPreferences(function ifKeyserverListIsInvalid_checkAgainInAnHour() {
   });
 }));
 
-test(withLogFiles(withPreferences(function keyRefreshServiceIsTurnedOffByDefault() {
+test(withLogFiles(withPreferences(function keyRefreshServiceIsTurnedOnByDefault() {
   const keyRefreshStartMessage = "keyRefreshService.jsm: Started";
   const keyserver = {};
 
   EnigmailKeyRefreshService.start(keyserver);
-  assertLogDoesNotContain(keyRefreshStartMessage);
+  assertLogContains(keyRefreshStartMessage);
 })));
 
 test(withLogFiles(withPreferences(function keyRefreshServiceStartsWhenPreferenceIsOn() {
